sdk/kusto/arm-kusto/package.json

```diff
@@ -98,5 +98,21 @@
     "docs": "echo skipped"
   },
   "sideEffects": false,
-  "autoPublish": true
+  "//metadata": {
+    "constantPaths": [
+      {
+        "path": "src/KustoManagementClient.ts",
```

Comment:
I think there's a bug in the code generator you are using. the real file name should be src/kustoManagementClient.ts. it has been fixed in latest code generator alpha release.

Reply:
done